tags/README.md

```diff
@@ -1,86 +1,216 @@
+  * White papers, presentations, videos, or other forms of training clarifying terminology, comparisons of different approaches, available projects or products, common or recommended practises, trends, illustrative successes and failures, etc.
+  * As far as possible, information should be based on research and fact gathering, rather than pure marketing or speculation.  
+
+#### End User Input Gathering (Inbound Communication)
+
+* Gather useful end user input and feedback regarding expectations, pain points, primary use cases etc.
+* Compile this into easily consumable reports and/or presentations to assist projects with feature design, prioritization, UX etc.
+
+#### Community Enablement
+
+* TAGs are open organizations with meetings, meeting agendas and notes, mailing lists, and other communications in the open
+* The mailing list, TAG meeting calendar, and other communication documents of the TAG will be openly published and maintained
+
+#### As Trusted Expert Advisors to the TOC
+
+* Perform technical due diligence on new and graduating projects, and advise TOC on findings.
+* Be involved with, or periodically check in with projects in their area, and advise TOC on health, status and proposed actions (if any) as necessary or on request.
+
+#### TAG Charter:
+
+* This is formally reviewed annually, and approved by the TOC.  The charter must clearly articulate:
+  * what is in and out of scope of the TAG,
+  * whether and how it overlaps and interfaces with other CNCF TAG’s or other relevant groups, and 
+  * how it operates and is governed, and specifically whether and how it deviates from standard TAG operating guidelines provided by the TOC.  Deviation from these guidelines is discouraged, unless there are good and well-documented reasons for such divergence, approved by the TOC. 
+
+See [Example Responsibilities of a CNCF TAG](https://docs.google.com/document/d/1L9dJl5aBFnN5KEf82J689FY0UtnUawnt9ooCq8SkO_w/edit?usp=sharing).
+
+## Operating Model
+
+Important: Each TAG is supported by a named member of the CNCF executive staff who is accountable for liaison with the CNCF Executive Director, plus communication and performance of the TAG, with quarterly and annual reporting to Governing Board & TOC.
+
+As a starting point let’s be inspired by CNCF OSS Projects and by K8s SIGs.  That means minimal viable governance and community-based organisation.
+
+### TAG Formation, Leadership and Membership Composition
+
+1. TAGs are formed by the TOC.  Initial TAGs are listed in [proposed TAGs](/tags/proposed.md), and will be adapted over time as required.  If members of the community believe that additional TAGs are desired, they should propose these to the TOC, with clear justification, and ideally volunteers to lead the TAG. The TOC wishes to have the smallest viable number of TAGs, and for all of them to be highly effective (as opposed to a "TAG sprawl" with large numbers of relatively ineffective TAGS).
```

Comment:
please update to reflect the comment from this issue: https://github.com/cncf/tag-security/issues/1206#issuecomment-1875877434

Reply:
Updated, ready for review.